src/components/Login/LoginModal.component.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Fade from '@material-ui/core/Fade';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Alert from '@material-ui/lab/Alert';
```

Comment:
It is better that you create this component with styled components. It will help you practice and you won't be adding a new dependency to your project.